Src/Base/AMReX_FabArrayCommI.H

```diff
@@ -275,7 +303,7 @@ FabArray<FAB>::ParallelCopy (const FabArray<FAB>& src,
         // we're doing plus()s on cell-centered data.  Don't do plus()s on
         // non-cell-centered data this simplistic way.
         //
-#ifdef AMREX_USE_OMP
+#ifdef _OPENMP
```

Comment:
This change should be reverted.  We prefer `AMREX_USE_OMP` to `_OPENMP`.  See a PR early on this subject.